MapboxNavigation/InstructionPresenter.swift

```diff
@@ -38,12 +38,12 @@ class InstructionPresenter {
         guard let source = self.dataSource else { return [] }
         var attributedPairs = self.attributedPairs(for: instruction, dataSource: source, imageRepository: imageRepository, onImageDownload: completeShieldDownload)
         let availableBounds = source.availableBounds()
-        let totalWidth = attributedPairs.attributedStrings.map { $0.size() }.reduce(.zero, +).width
+        let totalWidth: CGFloat = attributedPairs.attributedStrings.map { $0.size() }.reduce(.zero, +).width
```

Comment:
Pretty sure this change doesn’t improve compile time in any measurable way, because the compiler still has to infer the type of the `map` call in order to reduce it and get the `width` of it.